bundles/org.openhab.io.homekit/pom.xml

```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>io.github.hap-java</groupId>
       <artifactId>hap</artifactId>
-      <version>2.0.4</version>
+      <version>2.0.5</version>
```

Comment:
Is it of concern that 2.0.4 is listed as latest [here](https://github.com/hap-java/HAP-Java/releases)?

Reply:
Not really. There's an issue with our GitHub action doing the release that it doesn't always create that for some reason. The artifact is on maven central through: https://mvnrepository.com/artifact/io.github.hap-java/hap